README.md

```diff
@@ -4,33 +4,25 @@
 [![Dependencies Status](https://david-dm.org/meanjs/generator-meanjs.svg)](https://david-dm.org/meanjs/generator-meanjs)
 
 MEAN.JS is a full-stack JavaScript open-source solution, which provides a solid starting point for [MongoDB](http://www.mongodb.org/), [Node.js](http://www.nodejs.org/), [Express](http://expressjs.com/), and [AngularJS](http://angularjs.org/) based applications. The idea is to solve the common issues with connecting those frameworks, build a robust framework to support daily development needs, and help developers use better practices while working with popular JavaScript components.
+
 ## Overview
 
 One of the most frequently asked features from MEAN users is a way to scaffold their applications. As we looked for a way to help the community build and deploy production level MEAN applications, we decided to go with a [Yeoman](http://yeoman.io/index.html) generator. Yeoman generators provides a powerful, easy to maintain, and open solution for scaffolding applications.
 
-
-
 So, we set out to work on the features we thought every developer needs, and we are proud to present the official MEAN.JS Yeoman generator. It includes a set of simple tools you can use to make your MEAN application development easier and way more fun.
 
-
-
 ## Getting Started
 
-Before you begin make sure you have the [yo scaffolding tool](http://yeoman.io/generators/) installed(As it is part of the Yeoman tool set you might have installed it before). To globally install *yo* you will need to use npm:
-
+Before you begin make sure you have the [yo scaffolding tool](http://yeoman.io/generators/) installed (As it is part of the Yeoman tool set you might have installed it before). To globally install *yo* you will need to use npm:
```

Comment:
Doesn't this link http://yeoman.io/learning/index.html provide a little bit more relevant info?